pkg/runctl/controller.go

```diff
@@ -38,6 +38,8 @@ const (
 	// It is an invalid Kubernetes name to avoid conflicts with real
 	// pipeline runs.
 	heartbeatStimulusKey = "Heartbeat Stimulus"
+
+	waitingFailed = "waiting failed"
```

Comment:
The name of the constant doesn't reflect what it is about. Is it an error messare, a key, ...?

Reply:
done